Vic2ToHoI4/Data_Files/configurables/GovernmentMappings.txt

```diff
@@ -0,0 +1,412 @@
+## Main Forms
```

Comment:
Instructions; and why is leader ideology different from government one - where is it used and when.
Replace ruling_party with required_party or vic2_party so it's more clear.